make/release.mk

```diff
@@ -1,3 +1,49 @@
+## Set this as an environment variable to enable signing commands using cmrel.
+## Format should be:
+## projects/<project>/locations/<location>/keyRings/<keyring>/cryptoKeys/<keyname>/cryptoKeyVersions/<keyversion>
+##
+## @category Release
+CMREL_KEY ?=
+
+.PHONY: release
+# Build all release artifacts which might be run or used locally, except
+# for anything signed.
+release-artifacts: server-binaries cmctl kubectl-cert_manager helm-chart static-manifests all-containers
+
+.PHONY: release-signed
+# Same as `release`, except it also signs the Helm chart. Requires CMREL_KEY
+# to be configured.
+release-artifacts-signed: release-artifacts helm-chart-signature
+
+.PHONY: staged-release
+## Create a full release ready to be staged, including containers bundled for
+## distribution. Requires CMREL_KEY to be configured.
+##
+## @category Release
+release: release-signed release-manifests release-containers
+	$(MAKE) --no-print-directory bin/release/metadata.json
+
+# Set this as an environment variable to enable signing commands using cosign
+# Format should be any accepted by cosign; for GCP, use:
+# gcpkms://projects/<project>/locations/<location>/keyRings/<keyring>/cryptoKeys/<keyname>/versions/<keyversion>
+# CMREL_KEY ?= ""
```

Comment:
suggestion: this commit repeats the outdated comment which was removed in [this commit on master](https://github.com/cert-manager/cert-manager/commit/ccd03ccdd9e959842c3914a773bd90c10848a391)

Reply:
Oops, my bad. I hope I didn't miss anything else 😬